docs/components/movement-sensor/cameramono.md

```diff
@@ -92,9 +90,10 @@ Edit and fill in the attributes as applicable.
 
 ## Attributes
 
-| Name                       | Type   | Inclusion    | Description                                                                                                                                              |
-| -------------------------- | ------ | ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `camera`                   | string | **Required** | The `name` of the [camera](/components/camera/) you want to use for visual odometry.                                                                     |
+<!-- prettier-ignore -->
+| Name                       | Type   | Inclusion    | Description |
+| -------------------------- | ------ | ------------ | ----------- |
+| `camera`                   | string | **Required** | The `name` of the [camera](/components/camera/) you want to use for visual odometry. |
 | `motion_estimation_config` | object | **Required** | See [motionestimation.go in RDK](https://github.com/viamrobotics/rdk/blob/99f62a1640f4c267b744bdfc2924e9fd4f7a3c60/vision/odometry/motionestimation.go). |
```

Comment:
I don't see `motion_estimation_config` anywhere in `https://github.com/viamrobotics/rdk/blob/99f62a1640f4c267b744bdfc2924e9fd4f7a3c60/vision/odometry/motionestimation.go`. I know this is outside of your edit, but if you know what this does: ideally we define it inline within this table rather than directing users out.

If not, no worries: for another time.

Reply:
This is a draft page so not going to  worry about it. I only edited it so that I could take out the outdated image, and then ended up messing with the table for funsies, but not trying to actually edit content here since it's an unsupported or experimental draft.

Reply:
Oh, you're right! This was removed in DOCS-1009 in favor of Visual Odometry! Sorry, please do indeed ignore my comment!